lms/security.py

```diff
@@ -135,6 +135,7 @@ def get_policy(request: Request):
                 secret=request.registry.settings["email_preferences_secret"],
                 domain=request.domain,
                 email_preferences_service=request.find_service(EmailPreferencesService),
+                use_secure_cookie=not request.registry.settings["dev"],
```

Comment:
This works, but it is different than what I did in https://github.com/hypothesis/lms/blob/ee781b56b02ccd40fabc5aaa6500c684a2fba9cf/lms/session.py#L13. I'm guessing the approach here is preferred. If so then `sesssion.py` ought to be updated for consistency.

Reply:
Added a commit to make this consistent